appdata-fix.patch

```diff
@@ -0,0 +1,69 @@
+ <component type="desktop-application">
+   <id>org.kde.kalk</id>
++  <metadata_license>CC0-1.0</metadata_license>
++  <project_license>GPL-3.0-or-later</project_license>
++  <content_rating type="oars-1.1"/>
+   <name>Calculator</name>
+   <name xml:lang="ca">Calculadora</name>
+   <name xml:lang="ca-valencia">Calculadora</name>
+@@ -77,29 +80,30 @@
+   <developer_name xml:lang="x-test">xxKDE Communityxx</developer_name>
+   <metadata_license>CC0-1.0</metadata_license>
+   <description>
+-    <p>Kalk is a powerful calculator designed for mobile platform.</p>
+-    <p xml:lang="ca">El Kalk és una potent calculadora dissenyada per a plataformes mòbils.</p>
+-    <p xml:lang="ca-valencia">El Kalk és una potent calculadora dissenyada per a plataformes mòbils.</p>
+-    <p xml:lang="de">Kalk ist ein leistungsfähiger Rechner für Mobil-Plattformen.</p>
+-    <p xml:lang="en-GB">Kalk is a powerful calculator designed for mobile platform.</p>
+-    <p xml:lang="es">Kalk es una potente calculadora designada para la plataforma móvil.</p>
+-    <p xml:lang="eu">Kalk, mugikorretako plataformarako diseinatutako kalkulagailu ahaltsu bat da.</p>
+-    <p xml:lang="fi">Kalk on mobiilialustalle suunniteltu tehokas laskin.</p>
+-    <p xml:lang="fr">Kalk est une calculatrice puissante conçue pour les plate-formes mobiles.</p>
+-    <p xml:lang="hu">A Kalk egy hatékony számológép mobil platformokra tervezve.</p>
+-    <p xml:lang="id">Kalk adalah kalkulator hebat yang dirancang untuk platform seluler.</p>
+-    <p xml:lang="it">Kalk è una potente calcolatrice progettata per piattaforma mobile.</p>
+-    <p xml:lang="ko">Kalk는 모바일 플랫폼용 다기능 계산기입니다.</p>
+-    <p xml:lang="nl">Kalk is een krachtige rekenmachine ontworpen voor het mobiele platform.</p>
+-    <p xml:lang="nn">Kalk er ein avansert kalkulator laga for mobiltelefonar.</p>
+-    <p xml:lang="pl">Kalk jest rozbudowanych kalkulatorem stworzonym na urządzenia przenośne.</p>
+-    <p xml:lang="pt">O Kalk é uma calculadora poderosa desenhada para plataformas móveis.</p>
+-    <p xml:lang="pt-BR">Kalk é uma poderosa calculadora desenhada para a plataforma móvel.</p>
+-    <p xml:lang="ro">Kalk e un calculator puternic, proiectat pentru platforma mobilă.</p>
+-    <p xml:lang="sk">Kalk je silná kalkulačka zavrhnutá pre mobilnú platformu.</p>
+-    <p xml:lang="sl">Kalk je zmogljiv kalkulator zasnovan za mobilne platforme.</p>
+-    <p xml:lang="sv">Kalk är en kraftfull räknare konstruerad för mobilplattformar.</p>
+-    <p xml:lang="uk">Kalk — потужний калькулятор, який розроблено для мобільних платформ.</p>
+-    <p xml:lang="x-test">xxKalk is a powerful calculator designed for mobile platform.xx</p>
++    <p>Kalk is a convergent calculator for Plasma.</p>
+   </description>
++  <screenshots>
++    <screenshot>
++      <caption>Calculator Page</caption>
++      <image>https://cdn.kde.org/screenshots/kalk/kalk.png</image>
++    </screenshot>
++  </screenshots>
++  <recommends>
```

Comment:
We should put this on all of our apps :( I will take care of it upstream. @barthalion Should I try to add this change as patches to the maui/plasma mobile Flathub MR I made too? Or it is fine if they appear only with the 21.06 release we will publish next month?

Reply:
There was another tag of interest, `display_length` that I wasn't sure if we should start adopting for stating that it works on mobile screen sizes.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

Reply:
It's fine to add things later.